clang/tools/clang-scan-deps/ClangScanDeps.cpp

```diff
@@ -88,6 +88,7 @@ static bool DeprecatedDriverCommand;
 static ResourceDirRecipeKind ResourceDirRecipe;
 static bool Verbose;
 static bool PrintTiming;
+static bool NoPrintTimingHeader;
```

Comment:
Nit: even if the command-line argument is "-no" I would phrase this as `PrintTimingHeader` with default `true`.